quick_start.rst

```diff
@@ -36,6 +36,10 @@ and using {Project} on Mac and Windows machines
 are discussed in the `installation section of the admin guide
 <{admindocs}/installation.html>`__.
 
+If you have an existing version of {Project} installed from source, which
+you wish to upgrade or remove / uninstall, see the `installation section of the admin guide
```

Comment:
Change `see the` to `also see the` since the same link occurred just prior.

Reply:
thanks, I have updated the docs